pennylane/math/__init__.py

```diff
@@ -69,6 +69,7 @@
     dm_from_state_vector,
     fidelity,
     fidelity_statevector,
+    expectation_value,
```

Comment:
Are we able to get this function to show up in the docstrings [here](https://xanaduai-pennylane--4484.com.readthedocs.build/en/4484/code/qml_math.html)? @lillian542 can help with this.

Reply:
@ludmilaasb I *think* if you add `"expectation_value"` it to the `__all__` list further down in this file it will render in the docs there. But I'm having trouble getting my local docs build to work, so I haven't been able to verify.